client/StatusTable/components/StatusTable.js

```diff
@@ -0,0 +1,80 @@
+import React, {useEffect} from 'react'
+import {useSelector, useDispatch} from 'react-redux'
+import classNames from './scss/StatusTable.module.scss'
+import {tableLoad} from '../actions'
+import WeekRow from './WeekRow'
+import ClockSvg from './ClockSvg'
+import * as selectors from '../selectors'
+import getClassNames from '../../utilities/getClassnames'
+
+const StatusTable = (props) => {
+    const startSelected = useSelector(selectors.isStartSelected);
+    const workweeks = useSelector(selectors.workweeks);
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(tableLoad.trigger());
+    }, [startSelected, dispatch]);
+
+    if (!workweeks || !workweeks.length) {
+        // when TABLE_LOAD_FAILURE
+        return (null);
+    }
+
+    const headers = [
+        'День',
+        '№',
+        'Начало',
+        'Окончание',
+        'Время',
+        'Отчет'
+    ];
+
+    const {className} = props;
+    const tableClasses = getClassNames({[className]: true, [classNames['status-table']]: true});
+    return (
+        <table className={tableClasses}>
+            <colgroup className={classNames['status-table__col-day']} />
+            <colgroup className={classNames['status-table__col-number']}/>
+            <colgroup className={classNames['status-table__col-start']} />
+            <colgroup className={classNames['status-table__col-finish']}/>
+            <colgroup className={classNames['status-table__col-time']} />
+            <colgroup className={classNames['status-table__col-real-time']}/>
+
+            <thead>
+                <tr>
+                {headers.map(th => {
```

Comment:
let's pass modyfier along with cell title in the array, so you can avoid using switch there